src/Resources/translations/validators.de.xlf

```diff
@@ -1,79 +0,0 @@
-<?xml version="1.0"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="en" datatype="plaintext" original="file.ext">
```

Comment:
Moving translations to the library won't work magically. Symfony automatically registers translations from bundles only.
To get translations from the library, you need to hook them yourselves.

For Symfony 2.8+, this can be done through the config:

```yaml
framework:
    translator:
        paths:
            - some/absolute/path/to/folder
```

This means you can use your DI extension to prepend such config automatically.

If you don't do it, you must document it so that users register them manually (not a good DX though)